content/javascript/concepts/loops/loops.md

```diff
@@ -34,6 +34,8 @@ while (i < 5) {
   console.log(i);
   i++;
 }
+
+// Output: 0 1 2 3 4 5
 ```
 
```

Comment:
Let's actually add a separate \```shell snippet with the output between lines # 39 and # 41.

```
0
1
2
3
4
```

Reply:
Sure, I got you